src/components/TodoButtons/TodoButtons.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useLayoutEffect } from "react";
+import Button from "react-bootstrap/Button";
+import { filterTodos } from "../../store/reducers/UserSlice";
+import { useAppDispatch, useAppSelector } from "../../hooks/hooks";
+import s from "./TodoButtons.module.scss";
+
+const TodoButtons = () => {
+  const checkerValue = useAppSelector((state) => state.todos.checker);
+  const [click, setClick] = useState<Array<boolean>>([false, false, false]);
+
+  const dispatch = useAppDispatch();
+
+  const handleClick = (fieldName: string, strNum?: number): void => {
+    setClick(click.map((el, i, arr) => (i === strNum ? !el : false)));
+    typeof strNum !== "number" && setClick(click.map((el, i) => (i === 0 ? true : false)));
```

Comment:
Try to simplify - i === 0 ? true : false

Reply:
I think I simplified a bit